### PR TITLE
Fixes to What is Ethereum

### PR DESCRIPTION
**Pull request by hughlang**
Mostly spelling fixes. Also removed the broken paragraph/bullets that starts with "This chapter discusses the".

### CODE REVIEW

File: source/introduction/what-is-ethereum.rst [hunk 1]
```diff
@@ -2,13 +2,13 @@
...
+Blockchain technology is the technological basis of Bitcoin, first described by its mysterious author Satoshi Nakamoto in his white paper "Bitcoin: A Peer-to-Peer Electronic Cash System", published in 2008. While the use of blockchains for more general uses was already discussed in the original paper, it was not until a few years later that blockchain technology emerged as a generic term. A blockchain is a distributed computing architecture where every network node executes and records the same transactions, which are grouped into blocks. Only one block can be added at a time, and every block contains a mathematical proof that verifies that it follows in sequence from the previous block. In this way, the blockchain’s "distributed database" is kept in consensus across the whole network. Individual user interactions with the ledger (transactions) are secured by strong cryptography. Nodes that maintain and verify the network are incentivized by mathematically enforced economic incentives coded into the protocol.
```
Comment (hughlang) [inline source/introduction/what-is-ethereum.rst]: Replaced smart quotes with normal quotes

File: source/introduction/what-is-ethereum.rst [hunk 2]
```diff
@@ -17,42 +17,35 @@ What is Ethereum?
...
+These transaction fees are collected by the nodes that validate the network. These "miners" are nodes in the Ethereum network that receive, propogate, verify, and execute transactions. The miners then group the transactions – which include many updates to the "state" of accounts in the Ethereum blockchain – into what are called "blocks", and miners then compete with one another for *their* block to be the next one to be added to the blockchain. Miners are rewarded with ether for each succesful block they mine. This provides the economic incentive for people to dedicate hardware and electricity to Ethereum network.
```
Comment (hughlang) [inline source/introduction/what-is-ethereum.rst]: Changed short dashes - 
to long dashes –

File: source/introduction/what-is-ethereum.rst [hunk 3]
```diff
@@ -17,42 +17,35 @@ What is Ethereum?
...
+Just as in the Bitcoin network, miners are tasked with solving a complex mathematical problem in order to succesfully "mine" a block. This is known as a Proof of Work. Any computational problem that requires orders of magnitude more resources to solve algorithmically than it takes to verify the solution is a good candidate for proof of work. In order to discourage centralisation due to the use of specialised hardware (e.g. ASICs), as has occurred in the Bitcoin network, Ethereum chose a memory-hard computational problem. If the problem requires memory as well as CPU, the ideal hardware is in fact the general computer. This makes Ethereum's Proof of Work ASIC-resistant, allowing a more decentralized distribution of of security than blockchains whose mining is dominated by specialized hardware, like Bitcoin.
 
-Ethereum is perhaps best described as an ecosystem: the core protocol is supported by various pieces of infrastructure, code, and community that together make up the Ethereum project.
-
```
Comment (hughlang) [inline source/introduction/what-is-ethereum.rst]: This sentence moved to last paragraph. Seems to fit better there

File: source/introduction/what-is-ethereum.rst [hunk 4]
```diff
@@ -2,13 +2,13 @@
 A next generation blockchain
 =======================================================================
 
-Ethereum is an open blockchain platform that lets anyone build, and use, decentralized applications that run on blockchain technology. Like bitcoin, no one controls or owns Ethereum - it is an open-source project built by  many people around the world. But unlike the Bitcoin protocol, Etherem was designed to be adaptable and flexible. It is easy to create new applications on the Ethereum platform, and with the Homestead release, it is now safe for anyone to use those applications. 
+Ethereum is an open blockchain platform that lets anyone build and use decentralized applications that run on blockchain technology. Like Bitcoin, no one controls or owns Ethereum - it is an open-source project built by  many people around the world. But unlike the Bitcoin protocol, Ethereum was designed to be adaptable and flexible. It is easy to create new applications on the Ethereum platform, and with the Homestead release, it is now safe for anyone to use those applications.
```
Comment (hughlang) [inline source/introduction/what-is-ethereum.rst]: Misspelling Ethereum in the first paragraph just won't do

Comment (jjmstark) [inline source/introduction/what-is-ethereum.rst]: Oy vey, good find.

**Pull request closed without merging**